src/editors/editor.ts

```diff
@@ -1,6 +1,8 @@
 declare interface Editor {
   name: string;
+  displayName: string;
```

Comment:
Let's just have `name` be formatted like `displayName`, then remove `displayName`.

Reply:
I think name can be accepted as a kind of `key` and `displayName` to friendly show the editr name.

Reply:
I was thinking `name` should be called `id` or `key` and implemented on `Editor` parent class by replacing whitespace and lowercasing `name`?

Reply:
If the `key` property gets `displayName` and remove whitespace and lowercasing?